daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -848,6 +848,79 @@ func GetPendingTransactionsByAddress(ctx lib.ReqContext, w http.ResponseWriter,
 	SendJSON(response, w, ctx.Log)
 }
 
+// AssetInformation is an httpHandler for route GET /v1//account/{addr:[A-Z0-9]{%d}}/assets/{index:[0-9]+}
```

Comment:
1. typo : you have two slashes after the v1 instead of just one.
2. you're using the "add" variable here and "creator" in the swagger description. I'm not really picky here, but might be less error prone to use the same